hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/cmd/FeedCommProcessor.java

```diff
@@ -185,17 +187,25 @@ public void run() {
                         String messageString = ApiDeserializer.toHawkularFormat(message);
                         Buffer buffer = new Buffer();
                         buffer.writeUtf8(messageString);
-                        FeedCommProcessor.this.webSocket.sendMessage(WebSocket.PayloadType.TEXT, buffer);
+                        RequestBody requestBody = RequestBody.create(WebSocket.TEXT, buffer.readByteArray());
+                        FeedCommProcessor.this.webSocket.sendMessage(requestBody);
                     } else {
                         BinaryData messageData = ApiDeserializer.toHawkularFormat(message,
                                 messageWithData.getBinaryData());
-                        BufferedSink sink = FeedCommProcessor.this.webSocket
-                                .newMessageSink(WebSocket.PayloadType.BINARY);
-                        try {
-                            emitToSink(messageData, sink);
-                        } finally {
-                            sink.close();
-                        }
+
+                        RequestBody requestBody = new RequestBody() {
+                            @Override
+                            public MediaType contentType() {
+                                return WebSocket.BINARY;
+                            }
+
+                            @Override
+                            public void writeTo(BufferedSink bufferedSink) throws IOException {
+                                    emitToSink(messageData, bufferedSink);
+                            }
+                        };
+
+                        FeedCommProcessor.this.webSocket.sendMessage(requestBody);
```

Comment:
adding message in the hopes that github figures out this conversation is still valid :)